src/validation.cpp

```diff
@@ -853,16 +853,7 @@ static bool WriteBlockToDisk(const CBlock &block, FlatFilePos &pos,
 }
 
 Amount GetBlockSubsidy(int nHeight, const Consensus::Params &consensusParams) {
-    int halvings = nHeight / consensusParams.nSubsidyHalvingInterval;
-    // Force block reward to zero when right shift is undefined.
-    if (halvings >= 64) {
-        return Amount::zero();
-    }
-
-    Amount nSubsidy = 50 * COIN;
-    // Subsidy is cut in half every 210,000 blocks which will occur
-    // approximately every 4 years.
-    return ((nSubsidy / SATOSHI) >> halvings) * SATOSHI;
+    return 26 * LOGOS;
```

Comment:
Oh look at that. So much cleaner.

Reply:
The most important fix in  Bitcoin history!